### ISSUE TITLE
Release 4.6.300: what improvements since 4.5.700?

### ISSUE THREAD

**Issue by tfreitasleal** [MarimerLLC/cslaforum#136]
Hi Rocky,

There was a lot of work lately. One wonders whether there were a lot of bugs or a lot of improvements...

1) I think you can say 4.6.300 is the best available version of CSLA .NET.

2) Can you summarize the improvements since 4.5.700?

3) Since we are summarizing, what modules were removed from CSLA on 4.6.300?

**Comment by rockfordlhotka** [comment]
You can see the change logs for the three 4.6.x releases:

* [4.6.100](https://github.com/marimerllc/csla/issues?page=1&q=is%3Aissue+is%3Aclosed+milestone%3A4.6.100&utf8=%E2%9C%93)
* [4.6.200](https://github.com/marimerllc/csla/issues?page=1&q=is%3Aissue+is%3Aclosed+milestone%3A4.6.200&utf8=%E2%9C%93)
* [4.6.300](https://github.com/marimerllc/csla/issues?page=1&q=is%3Aissue+is%3Aclosed+milestone%3A4.6.300&utf8=%E2%9C%93)

There are some important non-technical or semi-technical enhancements, including:

* From version 4.6.300 CSLA is now under the MIT license
* Code of Conduct document for contributors and people participating in this forum
* CI server using Appveyor (and a private one using VSTS)
* Now using ZenHub to better track/manage issues and backlog
* Attach gitter site to this forum for any real-time chats we all might want to enjoy

At a high level the technical enhancements include:

* Move all code to a unified set of shared projects - no more "dangling" code files in platform-specific folders, making maintenance easier and cross-platform consistency higher
* Add .NET 4.6 support
* Update to latest version of Xamarin
* Add .NET Core 1.0 support
* Add ASP.NET Core 1.0 support
* Add several analyzers for VS15 that help improve coding quality
* Add support for custom serialization formatters
* New `LazyGetProperty` and `LazySetProperty` methods to simplify lazy loading of values
* Support new `Serialized` and other implementations in Xamarin and UWP
* All platforms now support `HttpProxy` talking to an `HttpPortal` endpoint
* Various bug fixes

**Comment by tfreitasleal** [comment]
Hi Rocky.

This is a very good answer. If there was support for sticky issues, I guess this answer be one of them.

**Comment by rockfordlhotka** [comment]
I did merge this content into the [Version 4.6.300 release information](https://github.com/MarimerLLC/csla/releases/tag/v4.6.300).
